zanata-war/src/main/java/org/zanata/rest/service/AsynchronousProcessResourceService.java

```diff
@@ -156,6 +157,19 @@ public ProcessStatus startSourceDocCreationOrUpdate(final String idNoSlash,
                                                     // progress
     }
 
+    /**
+     * This is used by functional test ZanataRestCaller for testing purposes.
```

Comment:
Is this only used by functional test? If yes we need to put it in zanata-test-war module. But I think this is used by client as well.